lib/geocoder/lookups/olleh.rb

```diff
@@ -182,51 +182,46 @@ def query_url_params(query)
           EY: query.options[:end_y],
           RPTYPE: 0,
           COORDTYPE: Olleh.route_coord_types[query.options[:coord_type]] || 7,
-          PRIORITY: Olleh.priority[query.options[:priority]],
-          timestamp:  now
-       }
-       (1..3).each do |x|
+          PRIORITY: Olleh.priority[query.options[:priority]]
+        }
+        (1..3).each do |x|
           s = [query.options[:"vx#{x}"], query.options[:"vy#{x}"]]
           hash.merge!({ "VX#{x}" => s[0], "VY#{x}" => s[1]}) unless s[0].nil? && s[1].nil?
         end
-
-        JSON.generate(hash)
       when "convert_coord"
-        JSON.generate({
+        hash = {
           x: query.text.first,
           y: query.text.last,
           inCoordType: Olleh.coord_types[query.options[:coord_in]],
-          outCoordType: Olleh.coord_types[query.options[:coord_out]],
-          timestamp: now
-       })
+          outCoordType: Olleh.coord_types[query.options[:coord_out]]
+       }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.